src/utils/defineTransform.js

```diff
@@ -35,7 +35,10 @@ export default function defineTransform ({
       varDeclaration = path.parentPath.value.id.name;
       classPath = path.parentPath.parentPath.parentPath;
     // Handle default exports
-    } else if (path.parentPath && path.parentPath.value && path.parentPath.value.type === 'ExportDefaultDeclaration') {
+    } else if (path.parentPath && path.parentPath.value &&
+               path.parentPath.value.type === 'ExportDefaultDeclaration' ||
+               path.parentPath.value.type === 'Property'
```

Comment:
Perhaps this should be a separate `else` block? Combining it with default exports makes the comments a little confusing.

Reply:
Also without parentheses (`()`) around the or-clause (`||`) it will fail if path.parentPath.value is undefined (order of operations with logical AND / OR)

Reply:
@phillipskevin Like this it will be another `else` for the same logic, how about updating the comment?